pages.ru/common/nohup.md

```diff
@@ -0,0 +1,20 @@
+# nohup
+
+> Позволяет процессу продолжать работу после закрытия терминала.
+> Больше информации: <https://www.gnu.org/software/coreutils/nohup>.
+
+- Запустить процесс, который может выполняться в отвязке от терминала:
+
+`nohup {{команда}} {{аргументы_команды}}`
```

Comment:
The following change could be applied after the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/common/nohup.md) is updated:

```suggestion
`nohup {{команда}} {{аргумент1 аргумент2 ...}}`
```

Wrong syntax is used for the placeholder supporting zero or more items.

Reply:
This and two other suggestions are implemented in #9464